### ISSUE TITLE
Mega menu examples should have `data-dropdown-toggle` rather than `data-collapse-toggle`

### ISSUE THREAD

**Issue by pekka** [themesberg/flowbite#581]
The examples on the Mega Menu page use the `data-collapse-toggle` attribute which doesn't support closing the menu by clicking anywhere on the page.

From where I'm standing, the examples should be updated to using `data-dropdown-toggle`

**Comment by chadalderson** [comment]
100% - I just tried this and it is much better. Also using data-collapse-toggle with two dropdown menus side by side does not toggle of visibility of the other, resulting in the hidden element IDs stacking. 

data-collapse-toggle is the way.

**Comment by MangoMarcus** [comment]
Yeah this confused me as the Collapse component doesn't seem to be documented anywhere and I'm not sure I understand the difference 🤔

**Comment by ENate** [comment]
Hi @chadalderson I am trying to design a site using mega menus in Flowbite and noticed that it is not possible toggle their visibility. Do you think is is better to implement the a function to close any of the drop down menus if I either click outside or on the next drop down or adjacent navbar item? Thanks,
